docs/configuration.md

```diff
@@ -230,3 +251,7 @@ Generated keypair with encoded pubkey: ak_2D9REvQsrAgnJgmdwPq585D8YksJC8PSAA8Msc
 
 In the example the generated public key is `ak_2D9REvQsrAgnJgmdwPq585D8YksJC8PSAA8MscQdxinbkFC7rq`, but **do not use it in your config**!
 This is just an **example** value to show what public key format you should expect after running `bin/aeternity keys_gen` command.
+
+### Configuration Examples
```

Comment:
May I suggest just deleting this paragraph "Configuration Examples"? It does not add much.